backend/src/api/audit.ts

```diff
@@ -71,7 +71,7 @@ class Audit {
           if (fits) {
             lastFeeRate = Math.min(lastFeeRate, tx.effectiveFeePerVsize);
           }
-          if (tx.firstSeen == null || (now - (tx?.firstSeen || 0)) > PROPAGATION_MARGIN) {
+          if (tx.firstSeen === null || (now - (tx?.firstSeen || 0)) > PROPAGATION_MARGIN) {
```

Comment:
Afraid this and that other case might be `undefined` here, changing to `!==` will break it .. maybe  @mononaut  can confirm.

Reply:
yeah, non-strict null equality checks are usually deliberate.

can we change the `eqeqeq` linting rule to `["error", "always", {"null": "ignore"}]`?